src/inspect_ai/tool/_tools/_web_search/_tavily.py

```diff
@@ -0,0 +1,77 @@
+import os
+from typing import Awaitable, Callable
+
+import httpx
+from pydantic import BaseModel, Field
+from tenacity import (
+    retry,
+    retry_if_exception,
+    stop_after_attempt,
+    stop_after_delay,
+    wait_exponential_jitter,
+)
+
+from inspect_ai._util.error import PrerequisiteError
+from inspect_ai._util.httpx import httpx_should_retry, log_httpx_retry_attempt
+from inspect_ai.util._concurrency import concurrency
+
+
+class TavilySearchResult(BaseModel):
+    title: str
+    url: str
+    content: str
+    score: float
+
+
+class TavilySearchResponse(BaseModel):
+    query: str
+    answer: str | None = Field(default=None)
+    images: list[object]
+    results: list[TavilySearchResult]
+    response_time: float
+
+
+def tavily_search_provider(
+    num_results: int, max_connections: int
+) -> Callable[[str], Awaitable[str | None]]:
+    tavily_api_key = os.environ.get("TAVILY_API_KEY", None)
+    if not tavily_api_key:
+        raise PrerequisiteError(
+            "TAVILY_API_KEY not set in the environment. Please ensure ths variable is defined to use Tavily with the web_search tool.\n\nLearn more about the Tavily web search provider at https://inspect.aisi.org.uk/tools.html#tavily-provider"
```

Comment:
I think this will be a link to somewhere in tools-standard.html -- we should also update this doc w/ the new interface / provider scheme.